cmd/abapEnvironmentRunAUnitTest.go

```diff
@@ -452,28 +515,72 @@ type ObjectSet struct {
 
 //Set in form of packages and software components to be checked
 type Set struct {
```

Comment:
All three structures "Set", "BaseSet", ExclusionSet" seem to be the same. Is it necessary to have three separate structures?

Reply:
Since the ABAP Unit API provides these possible sets they have been included. However, with this PR the User will get a warning information if he uses one of these sets. Furthermore, these sets will not be included in any request.

Reply:
From an implementation perspective, those are the same struct with a different name. Two of them can be left out. Programatically, you check for the "Type" field.